Makefile.core.mk

```diff
@@ -384,7 +384,10 @@ copy-templates:
 		for profile in manifests/helm-profiles/*.yaml ; do \
 			sed "1s|^|$${warning}\n\n|" $$profile > manifests/charts/$$chart/files/profile-$$(basename $$profile) ; \
 		done; \
-		cp manifests/zzz_profile.yaml manifests/charts/$$chart/templates ; \
+		[[ "$$chart" == "ztunnel" ]] && flatten="true" || flatten="false" ; \
```

Comment:
As discussed, this effectively moves the problem (from istioctl munging to makefile munging) rather than fix it - but it'll do for #51943, for now.